doc/en/components/grids/_shared/keyboard-navigation.md

```diff
@@ -0,0 +1,337 @@
+> Pressing the <kbd>Tab</kbd> key now goes through the tab stops in the following order: **GroupBy** / **Toolbar** -> **Headers** -> **Body** -> **Summaries** -> **Footer / Paginator**.
+
+>[!NOTE]
+> Exposing any **focusable** element into the **{ComponentTitle}** body via template may introduce **side effects** in the keyboard navigation, since the default
+> browser behavior is not prevented. It is the developer's responsibility to prevent or modify it appropriately.
+
+## Header Navigation
+
+A full *keyboard navigation* support in the **{ComponentTitle}** header is now introduced. Column headers can be easily traversed with the arrow keys. Additionally, there are a number of key combinations that trigger actions on the columns like **filtering**, **sorting**, **grouping** and etc.
+When the **{ComponentTitle}** header container is focused, the following key combinations are available:
+
+### Key Combinations
+
+ - <kbd>Arrow Up</kbd> navigates one cell up in the headers (no looping). Available only when Multi-row Layout (MRL) or Multi-column Headers (MCH) are defined
+ - <kbd>Arrow Down</kbd> navigates one cell down in the headers (no wrapping). Available only when Multi-row Layout (MRL) or Multi-column Headers (MCH) are defined
+ - <kbd>Arrow Left</kbd> navigates one cell left (no looping)
+ - <kbd>Arrow Right</kbd> navigates one cell right (no wrapping between lines)
+ - <kbd>Ctrl + Arrow Left</kbd> navigates to the leftmost cell in the row; if MRL or MCH are enabled, navigates to the leftmost cell at the same level
+ - <kbd>Home</kbd> navigates to the leftmost cell in  the row; if MRL or MCH are enabled, navigates to the leftmost cell at the same level
+ - <kbd>Ctrl + Arrow Right</kbd> navigates to the rightmost cell in row; if MRL or MCH are enabled, navigates to the rightmost cell at the same level
+ - <kbd>End</kbd> navigates to the rightmost cell in row; if MRL or MCH are enabled, navigates to the rightmost cell at the same level
+ - <kbd>Alt + L</kbd> opens Advanced Filtering dialog if Advanced Filtering is enabled
+ - <kbd>Ctrl + Shift + L</kbd> opens the Excel Style Filter dialog or the default (row) filter if the column is filterable
+ - <kbd>Ctrl + Arrow Up</kbd> sorts the active column header in ASC order. If the column is already sorted in ASC, sorting state is cleared
+ - <kbd>Ctrl + Arrow Down</kbd> sorts the active column header in DSC order. If the column is already sorted in DSC, sorting state is cleared
+ - <kbd>Space</kbd> selects the column; If the column is already selected, selection is cleared
+ <!-- ComponentStart:Grid -->
+ - <kbd>Shift + Alt + Arrow Left</kbd> groups the column, if the column is marked as groupable
+ <!-- ComponentEnd:Grid -->
+ - <kbd>Shift + Alt + Arrow Right</kbd> ungroups the column, if the column is marked as groupable
+ - <kbd>Alt + Arrow Left</kbd> or <kbd>Alt + Arrow Up</kbd> collapses the column group header, if the header is not already collapsed
```

Comment:
in all above keyboard combination change:
<kbd>Ctrl + Shift + L</kbd>
to:
<kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>L</kbd>